doc/ImplementingApplications/Steps2IntegrateElasticsearch/Steps2IntegrateElasticsearch.md

```diff
@@ -0,0 +1,255 @@
+```
+{
+    "uuid": "alt-2-0-1-es-c-es-1-0-0-000",
+    "ltp-direction": "core-model-1-4:TERMINATION_DIRECTION_SINK",
+    "client-ltp": [
+    ],
+    "server-ltp": [
+        "alt-2-0-1-http-c-es-1-0-0-000"
+    ],
+    "layer-protocol": [
+        {
+        "local-id": "0",
+        "layer-protocol-name": "elasticsearch-client-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_ELASTICSEARCH_LAYER",
+        "elasticsearch-client-interface-1-0:elasticsearch-client-interface-pac": {
+            "elasticsearch-client-interface-configuration": {
+            "auth": {
+                "api-key": "API key not yet defined."
+            },
+            "index-alias": "alt-2-0-1"
+            },
+            "elasticsearch-client-interface-status": {
+            "operational-state": "elasticsearch-client-interface-1-0:OPERATIONAL_STATE_TYPE_NOT_YET_DEFINED",
+            "life-cycle-state": "elasticsearch-client-interface-1-0:LIFE_CYCLE_STATE_TYPE_NOT_YET_DEFINED"
+            }
+        }
+        }
+    ]
+}
+```
+
+**HTTP client**
+```
+{
+    "uuid": "alt-2-0-1-http-c-es-1-0-0-000",
+    "ltp-direction": "core-model-1-4:TERMINATION_DIRECTION_SINK",
+    "client-ltp": [
+        "alt-2-0-1-es-c-es-1-0-0-000"
+    ],
+    "server-ltp": [
+        "alt-2-0-1-tcp-c-es-1-0-0-000"
+    ],
+    "layer-protocol": [
+        {
+        "local-id": "0",
+        "layer-protocol-name": "http-client-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_HTTP_LAYER",
+        "http-client-interface-1-0:http-client-interface-pac": {
+            "http-client-interface-configuration": {
+            "application-name": "ElasticSearch",
+            "release-number": "1.0.0"
+            }
+        }
+        }
+    ]
+}
+```
+
+**TCP client**, where IP address and port of running ES should be configured. If more ES clients are present in config file, they need to be all added here under client-ltp array.
+```
+{
+    "uuid": "alt-2-0-1-tcp-c-es-1-0-0-000",
+    "ltp-direction": "core-model-1-4:TERMINATION_DIRECTION_SINK",
+    "client-ltp": [
+        "alt-2-0-1-http-c-es-1-0-0-000"
+    ],
+    "server-ltp": [
+    ],
+    "layer-protocol": [
+        {
+        "local-id": "0",
+        "layer-protocol-name": "tcp-client-interface-1-0:LAYER_PROTOCOL_NAME_TYPE_TCP_LAYER",
+        "tcp-client-interface-1-0:tcp-client-interface-pac": {
+            "tcp-client-interface-configuration": {
+            "remote-protocol": "tcp-client-interface-1-0:PROTOCOL_TYPE_HTTP",
+            "remote-address": {
+                "ip-address": {
+                "ipv-4-address": "1.1.3.15"
+                }
+            },
+            "remote-port": 3015
+            }
+        }
+        }
+    ]
+}
+```
+
+The file ElasticsearchService.js contains class ElasticsearchService, operationalStateEnum and some helper exported methods.
```

Comment:
Could you please include detail like where actually we are using the file ElasticsearchService.js.